### PR TITLE
added typescript test script with reference to jQuery

### PR DESCRIPTION
**Pull request by AleksandrSharykin**

### CODE REVIEW

File: src/Axe/wwwroot/js/site.js [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
 
+    //$(window).contextmenu(function (e) {
+    //    console.log(this);
+    //    //e.preventDefault();
+    //})
+
+    var pr = new Promise(function (resolve, reject) {
+        setTimeout(function () {
+            resolve('tick')
+        }, 2000);
```
Comment (SergeyConflicts) [inline src/Axe/wwwroot/js/site.js:11]: Why this timeout is so big? (2000)

File: src/Axe/wwwroot/js/site.js [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+    });
+
+    pr.then(function (res) { console.log(res) });
+    console.log('continue');
+
+    $('code').contextmenu(function (e) {
+        console.log(this);
+        //e.preventDefault();
```
Comment (SergeyConflicts) [inline src/Axe/wwwroot/js/site.js:19]: Do not leave commented code. Remove it if it's not needed anymore.

File: src/Axe/wwwroot/js/site.js [hunk 3]
```diff
@@ -1,5 +1,24 @@
 ﻿$(document).ready(function () {
 
+    //$(window).contextmenu(function (e) {
```
Comment (SergeyConflicts) [inline src/Axe/wwwroot/js/site.js:3]: remove this commented code

File: src/Axe/wwwroot/js/site.js [hunk 4]
```diff
@@ -1,5 +1,24 @@
...
+    //    console.log(this);
+    //    //e.preventDefault();
+    //})
+
+    var pr = new Promise(function (resolve, reject) {
```
Comment (SergeyConflicts) [inline src/Axe/wwwroot/js/site.js:8]: pr is not very helpful variable name here

**Review by SergeyConflicts — CHANGES_REQUESTED** [review]

File: src/Axe/wwwroot/js/site.js (hunk 1 above)
Comment (AleksandrSharykin) [reply src/Axe/wwwroot/js/site.js:11]: 2000 ms = 2 sec isn't big. it is short but noticeable delay

**Pull request closed without merging**